### ISSUE TITLE
zotero.desktop not working for Ubuntu LTS 24.04

### ISSUE THREAD

**Issue by nikitas-theo** [retorquere/zotero-deb#100]
Hello, I installed Zotero following the instructions, the applications starts from terminal but won't appear in the launcher as an Icon. I run ` desktop-file-validate zotero.desktop`, and there was an issue with the Description element. After deleting it I don't get anything from desktop-file-validate, but I still don't see anything in the launcher. 

Here is the error from zotero.desktop 
`zotero.desktop: error: file contains key "Description" in group "Desktop Entry", but keys extending the format should start with "X-"`

I also installed from the Zotero tar and again it wasn't working, so maybe it is a more general issue

**Comment by retorquere** [comment]
New packages are up that fix the validation error, but the validation error has never stopped the icon from showing up.

When you say "install from the tar", does that mean "extracted and ran", or did you follow the instructions from https://www.zotero.org/support/installation ? If the latter, I don't know what that changed on the filesystem, it may be the case that /usr/share/applications/zotero.desktop (which is what the deb installed) is no longer being looked for.

**Comment by retorquere** [comment]
Without further feedback I will have to close this issue.

**Comment by muety** [comment]
I'm getting a similar error. It says "Desktop file didn't specify Exec field".

**Comment by retorquere** [comment]
Have you installed the tarball alongside the deb?

**Comment by muety** [comment]
I had it installed previously, but then removed it and switched to the DEB.

**Comment by retorquere** [comment]
Please uninstall, purge, and reinstall the deb. I don't know what the tarball script does on disk. Is there a zotero.desktop file anywhere under your home directory?

**Comment by retorquere** [comment]
Try

```
find $HOME -type f -iname zotero.desktop
```

and see if that finds anything.

**Comment by retorquere** [comment]
I'm running `desktop-file-validate` in my test suite now and it's coming up clear. 

What does

```
ls $HOME/.local/share/applications/*.desktop
```

say?

**Comment by retorquere** [comment]
I'm running `desktop-file-validate` in my test suite now and it's coming up clear. 

What does

```
locate '*zotero*.desktop'
```

say?

**Comment by muety** [comment]
Got it sorted by deleting the `zotero.desktop` file at `~/.local/share/applications/`. Thanks!

**Comment by retorquere** [comment]
Yeah that must have come from the tarball install.

Since @nikitas-theo hasn't responded I'm going to consider this issue addressed.

**Comment by nikitas-theo** [comment]
Hello, thanks for addressing the issue. Sorry for not replying sooner, I didn't get any notification about the thread until your mention.  I did follow the instructions from  https://www.zotero.org/support/installation,  with the set_launcher icon and and symlink. 

Following @muety's comment I also deleted  `~/.local/share/applications/`, removed and re-installed this deb package and now it's working. 

Cheers!
